src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+import java.util.Random;
+
+public class Lottery {
+
+  Random random = new Random();
+  ColorSupplier colorSupplier = new ColorSupplier();
+
+  public Ball getRandomBall() {
+    int randomNumber = random.nextInt(100) + 1;
+    String randomColor = colorSupplier.getRandomColor();
+    Ball ball = new Ball(randomColor, randomNumber);
+    return ball;
+  }
```

Comment:
You can directly return the new Ball object without creating a temporary 'ball' variable as per the checklist item 'Don't create temporary variables if you can directly return value from the method'.